LICENSE

```diff
@@ -0,0 +1,6 @@
+Permission is hereby granted, free of charge, to any person obtaining a copy of this [Application] and associated documentation files, to deal in the [Application] without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the [Application], and to permit persons to whom the [Application] is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the [Application].
+
+THE [Application] IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE [Application] OR THE USE OR OTHER DEALINGS IN THE [Application].
+## Copyright 2023, Fatima Najfi, Ochieng Steven and Erick Bupilipili
```

Comment:
- [ ] Hello! 😊 While it's awesome to utilize custom templates, it's crucial to tailor them to precisely fit the requirements of your specific project. 👌 Take a moment to examine your LICENSE file closely, and let's give it a personalized touch in accordance with the project's needs. 📄💼 Pay meticulous attention to detail to steer clear of minor errors that can give the file an unprofessional appearance. 🧐🚀